geoips/commandline/test_interfaces.py

```diff
@@ -85,17 +32,15 @@ def main():
     failed_plugins_errors = []
     failed_plugins_tracebacks = []
 
-    out_dicts = {}
+    curr_interfaces = interfaces.__dict__.values()
+    if len(sys.argv) > 1:
+        curr_interfaces = []
+        for curr_interface in sys.argv[1:]:
+            curr_interfaces += [interfaces.__dict__[curr_interface]]
 
-    (
-        failed_plugins,
-        successful_interfaces,
-        successful_plugins,
-    ) = test_deprecated_interfaces(
-        failed_plugins, successful_interfaces, successful_plugins
-    )
+    out_dicts = {}
 
-    for curr_interface in interfaces.__dict__.values():
+    for curr_interface in curr_interfaces:
```

Comment:
@jsolbrig Please confirm this script is sufficient for validating plugins.
@evrose54 You can call this with the interface name, and it will just run on that interface.  Should tell you what is wrong within all the plugins in that interface.

We'll want to update this eventually to actually be a valid pytest - but if it does the job, I think we can keep it for now unless it is VERY easy to set up the pytest version.

Reply:
I tested this using a "bad" plugin where I removed the various different required attributes. It wasn't working correctly and allowed bad plugins to pass.

I've updated this to correctly check for `interface`, `family`, `name`, and a non-empty docstring.